### PR TITLE
feat: enable relay websocket subchart

### LINKED ISSUE #2162: need to enable web sockets for JSON RPC Relay
We are being requested to enable the web sockets for JSON RPC Relay.  It is a separate chart.  But, there is an umbrella chart.  So, we want to switch to using the umbrella chart, deploy the way we currently do.  Then the customer requesting will be able to provide the `ws.enabled: true` in a values file override.

the umbrella chart: https://github.com/hiero-ledger/hiero-json-rpc-relay/tree/main/charts/hedera-json-rpc

### PR DESCRIPTION
**Pull request by JeffreyDallas**
## Description

This pull request changes the following:

* Change relay chart to top level chart to include websocket sub chart
* Multiple places need to change lablels
* updated getRelayLabels function so it can consistently read label of both old and new charts

Old chart has labels:
```
│ Labels:           app=hedera-json-rpc-relay                                                                                                                                                                    
│                   app.kubernetes.io/instance=relay-node1-node2                                                                                                                                                
│                   app.kubernetes.io/name=hedera-json-rpc-relay                                                                                                                                                 
│                   pod-template-hash=6bc57f567b 
```
New chart has labels:
```
│ Labels:           app=relay                                                                                                                                                                                    
│                   app.kubernetes.io/instance=relay-node1                                                                                                                                                       
│                   app.kubernetes.io/name=relay                                                                                                                                                                 
│                   pod-template-hash=7cc9797df7 
│ Labels:           app=ws                                                                                                                                                                                       
│                   app.kubernetes.io/instance=relay-node1                                                                                                                                                       
│                   app.kubernetes.io/name=ws                                                                                                                                                                    
│                   pod-template-hash=77fdf5f544  
```

### Related Issues

* Closes #2162

### Pull request (PR) checklist

* \[ ] This PR added tests (unit, integration, and/or end-to-end)
* \[ ] This PR updated documentation
* \[ ] This PR added no TODOs or commented out code
* \[x] This PR has no breaking changes
* \[ ] Any technical debt has been documented as a separate issue and linked to this PR
* \[ ] Any `package.json` changes have been explained to and approved by a repository manager
* \[x] All related issues have been linked to this PR
* \[x] All changes in this PR are included in the description
* \[x] When this PR merges the commits will be squashed and the title will be used as the commit message, the 'commit message guidelines' below have been followed

### Testing

* \[ ] This PR added unit tests
* \[ ] This PR added integration/end-to-end tests
* \[ ] These changes required manual testing that is documented below
* \[ ] Anything not tested is documented

The following manual testing was done:

* TBD

The following was not tested:

* TBD

<details>
<summary>
Commit message guidelines
</summary>
We use 'Conventional Commits' to ensure that our commit messages are easy to read, follow a consistent format, and for automated release note generation. Please follow the guidelines below when writing your commit messages:

1. BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with MAJOR in Semantic Versioning). A BREAKING CHANGE can be part of commits of any type.  NOTE: currently breaking changes will only bump the MAJOR version.
2. The title is prefixed with one of the following:

| Prefix    | Description                                         | Semantic Version Update | Captured in Release Notes |
|-----------|-----------------------------------------------------|-------------------------|---------------------------|
| feat:     | a new feature                                       | MINOR                   | Yes                       |
| fix:      | a bug fix                                           | PATCH                   | Yes                       |
| perf:     | performance                                         | PATCH                   | Yes                       |
| refactor: | code change that isn't feature or fix               | none                    | No                        |
| test:     | adding missing tests                                | none                    | No                        |
| docs:     | changes to documentation                            | none                    | Yes                        |
| build:    | changes to build process                            | none                    | No                        |
| ci:       | changes to CI configuration                         | none                    | No                        |
| style:    | formatting, missing semi-colons, etc                | none                    | No                        |
| chore:    | updating grunt tasks etc; no production code change | none                    | No                        |

</details>

### CODE REVIEW

File: src/core/config/remote/remote-config-validator.ts [hunk 1]
```diff
@@ -35,12 +36,20 @@ export class RemoteConfigValidator implements RemoteConfigValidatorApi {
...
     //  Add logic for selecting by specific label,
     //  when multiple instances can be deployed at the same time.
-    return [constants.SOLO_RELAY_LABEL];
+    const relayComponent = component as RelayNodeStateSchema;
+    const ids: number[] = relayComponent.consensusNodeIds;
+    console.log(`relayComponent.consensusNodeIds = ${JSON.stringify(relayComponent.consensusNodeIds)}`);
+    // generate label such as relay-node1 or relay-node1-node2
+    let label: string = 'relay';
+    for (const id of ids) {
+      label += '-' + Templates.renderNodeAliasFromNumber(id + 1);
```
Comment (jeromy-cannon) [inline src/core/config/remote/remote-config-validator.ts:50]:
```suggestion
      label += `-${Templates.renderNodeAliasFromNumber(id + 1)}`;
```
consistency in how we join strings with variables and function return statements

File: src/core/config/remote/remote-config-validator.ts [hunk 2]
```diff
@@ -35,12 +36,20 @@ export class RemoteConfigValidator implements RemoteConfigValidatorApi {
...
-  private static getRelayLabels(): string[] {
+  private static getRelayLabels(component: BaseStateSchema): string[] {
     // TODO:
     //  https://github.com/hashgraph/solo/issues/1823
     //  Add logic for selecting by specific label,
     //  when multiple instances can be deployed at the same time.
-    return [constants.SOLO_RELAY_LABEL];
+    const relayComponent = component as RelayNodeStateSchema;
+    const ids: number[] = relayComponent.consensusNodeIds;
+    console.log(`relayComponent.consensusNodeIds = ${JSON.stringify(relayComponent.consensusNodeIds)}`);
```
[the remark this answers is not in the document]
Comment (JeffreyDallas) [reply src/core/config/remote/remote-config-validator.ts]: fixed

**Review by instamenta — APPROVED** [review]

**Review by jeromy-cannon — APPROVED** [review]
